jpw1991/chebs-necromancy
# 128 neckro forced store timercounter

# Description

- Neckros can now be configured to drop stuff off at forced intervals (as long as their inventory has something inside)

## Compiled version [here](https://github.com/jpw1991/chebs-necromancy/releases/tag/2.3.10)

**Attention:** assumes newest Valhiem version + upgraded BepInEx and Jotunn. May/may not work with old version

## Testing

- [ ] Normal functionality
  - Delete configuration file
  - Run game
  - Test that Neckro works as he normally does -> picking up items and returning them once his backpack is full (or unless there's nothing else to pick up)
- [ ] Forced drop off interval
  - Quit game
  - Find and modify the `MaxSecondsBeforeDropoff` option and increase it to something useful for testing purposes (recommended value: 10)
  - Start game
  - Observe Neckro's behavior. If working correctly: the Neckro will ignore items on the ground and favor returning whatever it's currently carrying as long as the time has elapsed
